controllers/comments.controllers.js

```diff
@@ -41,3 +42,20 @@ exports.postCommentByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.removeCommentById = (req, res, next) => {
+  const { comment_id: commentId } = req.params;
+
+  const idExists = checkExists(
+    "comments",
+    "comment_id",
+    commentId,
+    "No comment matching requested id"
+  );
+  return Promise.all([commentId, idExists])
```

Comment:
You don't need to contact the database using 2 models here! What to Postgres tell you if it hasn't deleted anything?